src/drive-clean.cpp

```diff
@@ -58,7 +58,10 @@ class AsyncWorker : public Nan::AsyncWorker {
       errorCode = GetLastError();
       sysCall = "IOCTL_DISK_DELETE_DRIVE_LAYOUT";
       SetErrorMessage("Couldn't delete drive layout");
-      return;
+    }
+
+    if (hDevice != INVALID_HANDLE_VALUE) {
```

Comment:
This test is not required, we already return if `hDevice` is an `INVALID_HANDLE_VALUE`: https://github.com/resin-io-modules/win-drive-clean/blob/03de5d949eccbee598b5ff8ffdcab44372bccb71/src/drive-clean.cpp#L50 .